test/lib/next-webdriver.ts

```diff
@@ -183,5 +183,12 @@ export default async function webdriver(
 
     console.log(`\n> Hydration complete for ${fullUrl}\n`)
   }
+
+  // This is a temporary workaround for turbopack starting watching too late.
+  // So we delay file changes by 500ms to give it some time
+  // to connect the WebSocket and start watching.
+  if (process.env.TURBOPACK) {
+    await waitFor(1000)
```

Comment:
Does the wait hydration logic above not work properly with turbopack mode or is it just not long enough of a delay alone?